src/azul/plugins/repository/tdr/__init__.py

```diff
@@ -0,0 +1,58 @@
+import logging
+from typing import (
+    List,
+    Sequence,
+)
+
+from deprecated import deprecated
+
+from azul.indexer import (
+    Bundle,
+    BundleFQID,
+)
+from azul.plugins import (
+    RepositoryPlugin,
+)
+from azul.tdr import (
+    AzulTDRClient,
+    BigQueryDataset,
+)
+from azul.types import (
+    JSON,
+    MutableJSONs,
+)
+
+log = logging.getLogger(__name__)
+
+
+class Plugin(RepositoryPlugin):
+
+    @classmethod
+    def get_dataset(cls) -> BigQueryDataset:
+        return BigQueryDataset('8c6633e3-ce9c-4aa4-b0ae-3fba122386a1', 'hca_mvp', False)
```

Comment:
If we make the project name configurable, we should also make these parameters configurable via the environment. To avoid proliferation of environment variables, we should compact multiple values into a single variable using an ARN-like syntax. Something like

`arn:google:bigquery:US:${project_id}:dataset/${dataset_name}`

The TDR endpoint should probably go into another variable but it looks like you don't need that here.

I'm confused why we need both name and ID. If we really do, we need to figure something out because the ARN syntax doesnt lend itself well to specifying conceptually redundant information.